hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/replication/regionserver/MetricsReplicationGlobalSourceSourceImpl.java

```diff
@@ -49,6 +50,7 @@
   private final MutableFastCounter completedRecoveryQueue;
   private final MutableFastCounter failedRecoveryQueue;
   private final MutableGaugeLong walReaderBufferUsageBytes;
+  private final MutableGaugeInt sourceInitializing;
```

Comment:
Nit:
so, this is the number of source initializing, right? 
mind adding a minor comment for the metrics?

Reply:
never mind, I see you have  comments below. :)